package/cloudshell/iac/terraform/services/tf_proc_exec.py

```diff
@@ -25,6 +25,14 @@ def __init__(self, shell_helper: ShellHelperObject, sb_data_handler: SandboxData
         self._sb_data_handler = sb_data_handler
         self._input_output_service = input_output_service
         self._tf_workingdir = sb_data_handler.get_tf_working_dir()
+        self._backend_handler = BackendHandler(
+            self._shell_helper.logger,
+            self._shell_helper.api,
+            self._shell_helper.attr_handler.get_attribute(ATTRIBUTE_NAMES.REMOTE_STATE_PROVIDER),
+            self._tf_workingdir,
+            self._shell_helper.sandbox_id,
+            self._sb_data_handler.get_tf_uuid()
+        )
```

Comment:
need to move this outside so that TfProcExec will get the BackendHandler as a dependency